composer.json

```diff
@@ -13,7 +13,8 @@
     "require-dev": {
         "php-vcr/phpunit-testlistener-vcr": "^1.1",
         "jakub-onderka/php-parallel-lint": "^0.9.2",
-        "jakub-onderka/php-console-highlighter": "^0.3.2"
+        "jakub-onderka/php-console-highlighter": "^0.3.2",
+        "phpunit/phpunit": "~4.8 || ~5.0"
```

Comment:
The fact is that this dependency has actually been properly resolved by composer. see: https://travis-ci.org/contentful/contentful.php/builds/215916416

Also this allows any people to test locally PR they submit to this repo, without forcing them to install globally a specific phpunit version